### ISSUE TITLE
TransGAN fid problem

### ISSUE THREAD

**Issue by Stephenlove** [asarigun/TransGAN#10]
hello,I would like to humbly ask you what is the difference beetween TransGAN-main and TransGAN-master?can Trans-main reproduce similar results of the original paper? The results obtained by using CIFAR in TransGAN-main are quite different from those in the paper,and WGAN-EP loss concussion,so I want to ask you.

**Comment by asarigun** [comment]
Hi @Stephenlove,

Thank you so much for being so interested in the repo! When I started to implement, I used [v2.0](https://arxiv.org/abs/2102.07074v2) of the paper. The results are different since the hyperparameter I used wasn't too many comparing to the original implementation. Also, there is limited computational power&GPU to train the model, and I have ongoing projects that's why I couldn't further develop performance. Therefore, the FID score is different from the reported results! However, qualitative results are reasonable, and it shows that it is open to developing the performance. I hope this could answer your question!
